elasticsearch.md

```diff
@@ -1487,63 +1488,166 @@ ReadonlyREST can write events very similarly to Logstash into to a series of ind
   }
 ```
 
-Here is a configuration example, you can see the `audit.collector: true` setting, which normally defaults to false. Note how the successful requests matched by the first rule \(Kibana\) will not be written to the audit log, because the verbosity is set to error. Audit log in facts, obey the verbosity setting the same way regular text logs do.
+#### Configuration
 
-**NB** Following audit configurations are presented with the separate `audit` section, although it can be omitted as settings defined using previous approach (defined directly under `readonlyrest` section and starting with `audit_`)` are still supported (but not recommended) preserving backward compatibility.
+The audit collecting by default is disabled. To enable it, you need to add `audit.enabled: true` and configure the `audit.outputs`.
+With the `outputs` array, you manage the audit sinks configurations.
+You can configure multiple outputs for audit events, but at least one audit output must be defined when the audit is enabled.
+The supported output types are:
+* `index` - similarly to Logstash it writes audit events in the documents stored in the indices named by default `readonlyrest_audit-YYYY-MM-DD`
+* `log` - it allows you to collect audit logs in a similar way to the app logs and format them with the help of features that `log4j2` enables.
+  Uses a dedicated logger from the elasticsearch's `log4j2` config.
 
 ```text
 readonlyrest:
 
-    audit:
-      collector: true
+  audit:
+    enabled: true
+    outputs: 
+    - type: index
 
-    access_control_rules:
+  access_control_rules:
 
-    - name: Kibana
-      type: allow
-      auth_key: kibana:kibana
-      verbosity: error
+   - name: Kibana
+     type: allow
+     auth_key: kibana:kibana
+     verbosity: error
 
-    - name: "::RO::"
-      auth_key: simone:ro
-      kibana_access: ro
+   - name: "::RO::"
+     auth_key: simone:ro
+     kibana_access: ro
 ```
 
-### Extended audit
+If you want to use the default settings of audit outputs, you can use the simple format of `audit.outputs`:
```

Comment:
IMO we should treat this as a default. The simple version will be probably the most commonly used. The extended version is for superusers.